### ISSUE TITLE
Doubt about the defined hinge version Loss

### ISSUE THREAD

**Issue by hhguo** [descriptinc/melgan-neurips#34]
Hi, I am curious about the hinge loss defined in your paper. It seems that your implementation is different.

This is the definition in your paper.
![image](https://user-images.githubusercontent.com/28179486/92690223-35587680-f373-11ea-8bd9-6d238577be22.png)

But you use F.relu in the code, which is max(0, D(x))
![image](https://user-images.githubusercontent.com/28179486/92690324-6173f780-f373-11ea-8fee-4d0b8d74dcce.png)
 
I tend to believe that the code implementation is correct, so maybe the paper definition is wrong?

**Comment by allenhung1025** [comment]
I have the same doubt, too.

**Comment by jamesliu** [comment]
I believe the implementation is right, but the paper definition is likely not correct(max instead of min or adding -). May someone confirm it? Thanks.
